rc/lsp.kak

```diff
@@ -1526,6 +1537,7 @@ map global lsp d '<esc>: lsp-definition<ret>'             -docstring 'go to defi
 map global lsp e '<esc>: lsp-diagnostics<ret>'            -docstring 'list project errors, info, hints and warnings'
 map global lsp f '<esc>: lsp-formatting<ret>'             -docstring 'format buffer'
 map global lsp h '<esc>: lsp-hover<ret>'                  -docstring 'show info for current position'
+map global lsp H '<esc>: lsp-hover-in-hover-client<ret>'   -docstring 'show info for current position in separate client'
```

Comment:
should add a line for this in README (in the "Either way you get:" list, next to `lsp-hover`)

(also alignment is off - I'll try to fix it for the entire block at some point)